API.js

```diff
@@ -6,7 +6,9 @@ import { ApolloProvider } from "@apollo/react-hooks";
 import _ from "lodash";
 
 export function APIClient() {
-  const httpLink = new HttpLink({ uri: "http://localhost:4000/graphql" });
+  const httpLink = new HttpLink({
+    uri: "https://mysterious-everglades-44359.herokuapp.com/graphql",
```

Comment:
Lets not have this URL hard coded. Can you please move it to a file we gitignore? We will probably end up adding more stuff to it later too. 

Looks like it can be done in 2 ways:

1. https://stackoverflow.com/questions/58270059/react-native-expo-environment-variables
2. https://www.freecodecamp.org/news/how-to-gracefully-use-environment-variables-in-a-react-native-app/

Reply:
good point anand